scripts/pre-release.sh

```diff
@@ -22,7 +22,7 @@ NEXT="${NEXT}-SNAPSHOT"
 echo "This script will:"
 echo "================="
 echo "Tag $CURRENT"
-echo "Update Narayana $BRANCH to $NEXT"
+echo "Update Narayana '$BRANCH' to project.version $NEXT"
```

Comment:
This is the only outstanding question really. Does it cause some problem without it? I don't quite follow what the purpose is sorry

Reply:
not necessary. removed.

context of the change: I had a trouble to understand what the script goes to do (aka. information `Update Narayana 5.9 to 5.9.10.Final-SNAPSHOT` sounded strange to me and I needed to go to the script and check what's going to be done) so I changed it to be more descriptive (from my perspective).